<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Action</class>
 <widget class="QWidget" name="Action">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Action</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startLoadButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>start load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopLoadButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>stop load</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="isLoadedCheckBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>load?</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="isSavedCheckBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>save?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopSaveButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>stop save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startSaveButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>start save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="freqLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="numOfSensLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
